src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -838,12 +838,12 @@ public String getQuestionTypeChoiceOption() {
         }
         */
 
-        for (String subQn : rubricSubQuestions) {
+        /*for (String subQn : rubricSubQuestions) {
```

Comment:
Commenting out code entire blocks of code without any strong reason is not a good idea.